<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="counterLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startCounterPushButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>203</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopCounterPushButton">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>203</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetCounterPushButton">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>203</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="soundBox">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>203</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSpinBox" name="hourSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>105</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minuteSpinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>105</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="secondSpinBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>105</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="hours">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>165</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Hours</string>
    </property>
   </widget>
   <widget class="QLabel" name="minutes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>165</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutes</string>
    </property>
   </widget>
   <widget class="QLabel" name="seconds">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>165</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
